pulsar-client/src/main/java/org/apache/pulsar/client/impl/TableViewImpl.java

```diff
@@ -364,8 +364,8 @@ private void readAllExistingMessages(Reader<T> reader, CompletableFuture<Void> f
                                   }
                                }).exceptionally(ex -> {
                                    if (ex.getCause() instanceof PulsarClientException.AlreadyClosedException) {
-                                       log.error("Reader {} was closed while reading existing messages.",
-                                               reader.getTopic(), ex);
+                                       log.info("Reader {} was closed while reading existing messages.",
```

Comment:
Why change the log level from error to info?

Reply:
To optimize the error logs because the `AlreadyClosedException` is an expected exception but the full stack will be printed each time